rust/bin-shared/Cargo.toml

```diff
@@ -7,12 +7,14 @@ description = "Firezone-specific modules shared between binaries."
 
 [dependencies]
 anyhow = "1.0.82"
-clap = { version = "4.5.4", features = ["derive"] }
+axum = { version = "0.7.5", default-features = false, features = ["http1", "tokio"] }
```

Comment:
Hm so the Gateway and Relay both need this, but the Client will be compiling a web server it doesn't need. And it'll show up in `cargo tree` as a dep. I hope that doesn't slow anything down. Tokio and Tauri are probably still the worst offenders for slow compiles

Reply:
We can feature gate that but it means compiling two versions of `bin-shared` instead. We'd have to profile what is quicker.